client/src/components/link-container/link-container.tsx

```diff
@@ -0,0 +1,27 @@
+import css from './link-container.module.css'
+
+function LinkContainer() {
+	return (
+		<div className={css.linkContainer}>
+			<a
+				href="https://www.reddit.com/r/HermitCraft/comments/10wksaw/hctcg_online/"
+				target="_blank"
+				rel="noreferrer"
+			>
+				<img src="/images/icons/reddit.svg" height="26" />
```

Comment:
Since we no longer have the text I would at least add title attribute for the links (Reddit/Github/Discord)